.github/workflows/flowzone.yml

```diff
@@ -11,12 +11,7 @@ jobs:
   flowzone:
     name: Flowzone
     uses: product-os/flowzone/.github/workflows/flowzone.yml@master
-    secrets:
-      FLOWZONE_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
-      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-      BALENA_API_KEY: ${{ secrets.BALENA_TOKEN }}
-      CUSTOM_JOB_SECRET_1: ${{ secrets.BALENA_STAGING_TOKEN }}
+    secrets: inherit
```

Comment:
This won't work until we have gh enterprise and `product-os` is within the same enterprise